src/excmds.ts

```diff
@@ -2331,36 +2331,27 @@ export async function tabclose(...indexes: string[]) {
     return done
 }
 
-/** Close all tabs to the right of the current one
+/** Close all tabs to the side specified
  *
  */
 //#background
-export async function tabclosealltoright() {
```

Comment:
To reinstate these commands, in src/lib/config.ts, under exaliases, add `tabclosealltoright: "tabcloseallto right", tabclosealltoleft: "tabcloseallto left"`.